app/ModelSerializers/Summit/AdminSummitEventSerializer.php

```diff
@@ -22,6 +22,9 @@ class AdminSummitEventSerializer extends SummitEventSerializer
 {
     protected static $array_mappings = [
         'Occupancy' => 'occupancy:json_string',
+        'OverflowStreamingUrl' => 'overflow_streaming_url:json_string',
```

Comment:
please review implementation
this fields are missing from allowed fields and in_array test

Reply:
done

Reply:
@romanetar is missing this part
https://github.com/OpenStackweb/summit-api/blob/e8ab6755771f18e95ab50c55baac1719cdfe6a90/app/ModelSerializers/Summit/AdminSummitEventSerializer.php#L72